scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2008. All rights reserved.
+#
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
+
+CHAIN=RH-Firewall-1-INPUT
+OP=$1
```

Comment:
This should be quoted in case `$1` contains a space: `OP="$1"` While not strictly necessary, the right-hand side of `CHAIN` should be also quoted.